src/aliceVision/sfm/pipeline/sequential/ReconstructionEngine_sequentialSfM.cpp

```diff
@@ -1272,18 +1288,16 @@ bool ReconstructionEngine_sequentialSfM::FindNextImagesGroupForResection(
 
 /**
  * @brief Add one image to the 3D reconstruction. To the resectioning of
- * the camera and triangulate all the new possible tracks.
+ * the camera.
  * @param[in] viewIndex: image index to add to the reconstruction.
+ * @param[out] resectionData: contains the pose all the data used during the resection.
```

Comment:
`resectionData: contains the camera pose and all data used during the resection.`